src/test/java/redis/clients/jedis/mocked/pipeline/PipeliningBaseBitmapCommandsTest.java

```diff
@@ -0,0 +1,264 @@
+package redis.clients.jedis.mocked.pipeline;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+
+import org.junit.Test;
+import redis.clients.jedis.Response;
+import redis.clients.jedis.args.BitCountOption;
+import redis.clients.jedis.args.BitOP;
+import redis.clients.jedis.params.BitPosParams;
+
+public class PipeliningBaseBitmapCommandsTest extends PipeliningBaseTestBase {
```

Comment:
You are putting these in pipeline package. So something other than pipeline should start these classes names. For example: `<DataStructure>MockedPipeliningTest`.

Reply:
See my response to the other similar comment about UnifiedJedis. Additionally here, I find it relevant that the subject under test is the `PipeliningBase` specifically. This should be emphasized by the naming. There is more to Pipelining, and it's not covered by this class.